catalog/video/catalog-mpegts-stream/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -52,6 +52,11 @@
                 name="Location Subsample Count" id="klvLocationSubsampleCount" required="true"
                 type="Integer" default="50"/>
 
+        <AD
+                description="Metacard Update Initial Delay (milliseconds)"
```

Comment:
How about:  The metacard updates need be delayed to compensate for the time SOLR takes to make new metacards available. Increase this delay to reduce the associated retry log messages. (seconds)